sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinModelData.java

```diff
@@ -0,0 +1,199 @@
+package com.azure.digitaltwins.core.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.digitaltwins.core.implementation.models.ModelData;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+/**
+ * The DigitalTwinModelData representing the model and its corresponding metadata.
+ */
+@Fluent
+public class DigitalTwinModelData {
+
+    private final ObjectMapper mapper;
+
+    /**
+     * Creates a new instance of DigitalTwinModelData.
+     * @param modelData The model data.
+     */
+    public DigitalTwinModelData(ModelData modelData) {
+        mapper = new ObjectMapper();
+
+        this.setId(modelData.getId())
+            .setUploadTime(modelData.getUploadTime())
+            .setDisplayName(modelData.getDisplayName())
+            .setDescription(modelData.getDescription())
+            .setDecommissioned(modelData.isDecommissioned())
+            .setModel(mapper.convertValue(modelData.getModel(), String.class));
+    }
+
+    /*
+     * A language map that contains the localized display names as specified in
+     * the model definition.
+     */
```

Comment:
I don't think we need java doc comments on these private fields, the getters and setters have the comments already, so this feels repetitive. What do you think?

Reply:
This is a copy of the ModelData, I tried to keep changes as minimal as possible.